.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Release .deb files'
+on: release
+
+jobs:
+  release-matrix:
```

Comment:
-matrix sounds verbose to me (please check the style in which current actions are displayed)

Reply:
removed.